src/main/java/com/faforever/client/remote/domain/GameType.java

```diff
@@ -0,0 +1,47 @@
+package com.faforever.client.remote.domain;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.lang.invoke.MethodHandles;
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+
+public enum GameType {
+
+  UNKNOWN("unknown"),
+  CUSTOM("custom"),
+  MATCHMAKER("matchmaker"),
+  COOP("coop"),
+  TUTORIAL("tutorial");
+
+  private static final Logger logger = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
```

Comment:
@SlF4j not working on enums?

Reply:
No idea I just copy and pasted from GameStatus

Reply:
Try use the annotation